features/step_definitions/catalogue_steps.rb

```diff
@@ -64,6 +64,10 @@
   puts "Noted page_count: #{@page_count}"
 end
 
+Then(/^I limit the search results to the (.*) category$/) do |category|
```

Comment:
Good stuff - could you just leave a comment here to explain _why_ we've got this special case?

Reply:
Done